frontend/src/components/enroll/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
```

Comment:
Is this file part of the ticket and PR? If so, please update ticket and PR, otherwise lets remove this file from PR.